tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -242,7 +242,7 @@ Verify newly published release is on Find Statistics page
 Navigate to published release page
     [Tags]  HappyPath
     user clicks testid element   View stats link for ${PUBLICATION_NAME}
-    user waits until h1 is visible   ${PUBLICATION_NAME}  90
+    user waits until h1 is visible   ${PUBLICATION_NAME}  360
```

Comment:
I think the service shouldn't be so slow such that a release page doesn't load in 90 seconds. If this test is failing due to it not loading in 90 seconds, it is an issue we should be raising.

So I'd keep the timeout at 90 seconds and if it happens again, raise it with others that this is happening.